docs/guides/react-components/install-plain-java.md

```diff
@@ -8,8 +8,169 @@ For this installation, you must have:
 - OpenJDK 21+ locally installed
 - Camunda `8.6.0-alpha2` or later
```

Comment:
I'm not sure if we need Camunda as a prereq if we're going to direct them to install it below - this might be fine with just OpenJDK as the requirement.

Reply:
I prefer to be extra explicit until this is GA'd (and not only available in specific alphas).